src/cli/init.ml

```diff
@@ -9,27 +9,27 @@ let write_file : string -> (formatter -> unit) -> unit = fun fn pp ->
   pp ppf; pp_print_flush ppf (); close_out oc
 
 let pp_makefile : formatter -> unit = fun ppf ->
-  fprintf ppf
-    ".POSIX:\n\
-     SRC = \n\
-     OBJ = $(SRC:.lp=.lpo)\n\
-     .SUFFIXES:\n\
-     \n\
-     all: $(OBJ)\n\
-     \n\
-     install: $(OBJ) %s\n\
-     \tlambdapi install %s $(OBJ) $(SRC)\n\
-     \n\
-     uninstall:\n\
-     \tlambdapi uninstall lambdapi.pkg\n\
-     \n\
-     clean:\n\
-     \trm -f $(OBJ)\n\
-     \n\
-     .SUFFIXES: .lp .lpo\n\
-     \n\
-     .lp.lpo:\n\
-     \tlambdapi check --gen-obj $<\n" Package.pkg_file Package.pkg_file
+  fprintf ppf "\
+.POSIX:
+SRC =
+OBJ = $(SRC:.lp=.lpo)
+.SUFFIXES:
+
+all: $(OBJ)
+
+install: $(OBJ) %s
+	lambdapi install %s $(OBJ) $(SRC)
+
+uninstall:
+	lambdapi uninstall lambdapi.pkg
+
+clean:
+	rm -f $(OBJ)
+
+.SUFFIXES: .lp .lpo
+
+%%.lpo: %%.lp
```

Comment:
The previous syntax `.lp.lpo: ...` is POSIX makefile syntax, while the one you're using here (with `%`) is GNU makefile syntax. IMO it's better if we keep POSIX syntax (https://pubs.opengroup.org/onlinepubs/009695299/utilities/make.html).

Reply:
I prefer the GNU Makefile syntax, it makes more sense; it's a shame that the POSIX syntax is the most widely recognized